docs/reference/geometry.md

```
@@ -7,7 +7,44 @@
 
-::: soundevent.geometry
+    or, if you are using uv
+
+    ```bash
+    uv add "soundevent[geometry]"
+    ```
+
+The `soundevent.geometry` module provides a comprehensive set of tools for handling the spatial and temporal aspects of sound events. It is organised into several components:
+
+*   [**Matching**](#matching): Tools for pairing sets of geometries.
+*   [**Affinity**](#affinity_similarity): Functions to calculate similarity scores (e.g., IoU) between geometries.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)